editors/sc-ide/CMakeLists.txt

```diff
@@ -101,6 +112,10 @@ set ( ide_moc_hdr
     widgets/util/color_widget.hpp
     widgets/util/docklet.hpp
     widgets/util/volume_widget.hpp
+    widgets/util/volume_widget.hpp
```

Comment:
There's a repeated element in this list.

Reply:
fixed